checker/src/summaries.rs

```diff
@@ -42,6 +42,11 @@ use std::ops::Deref;
 ///    with explicit contract functions.
 #[derive(Serialize, Deserialize, Clone, Debug, Default, Eq, PartialEq, Hash)]
 pub struct Summary {
+    /// The number of seconds that the analyzer used to construct this summary.
+    /// If it is >= k_limits::MAX_ANALYSIS_TIME_FOR_BODY then this summary is not definitive
+    /// because the analyzer will omit it from the outer fixed point loop.
+    pub analysis_time_in_seconds: u64,
```

Comment:
Any particular reason you want to use u64 instead of something like Duration?

Reply:
Duration seems a bit like overkill and it does not support a default formatter, but it is a fair point. I keep it in mind for the future.